go.mod

```diff
@@ -1,8 +1,12 @@
 module contrib.go.opencensus.io/exporter/prometheus
 
 require (
-	github.com/prometheus/client_golang v0.9.3
-	github.com/prometheus/common v0.4.1 // indirect
-	github.com/prometheus/procfs v0.0.1 // indirect
-	go.opencensus.io v0.22.0
+	github.com/cespare/xxhash/v2 v2.1.1 // indirect
+	github.com/golang/groupcache v0.0.0-20191027212112-611e8accdfc9 // indirect
+	github.com/prometheus/client_golang v1.2.1
+	github.com/prometheus/procfs v0.0.6 // indirect
+	go.opencensus.io v0.22.2
+	golang.org/x/sys v0.0.0-20191113165036-4c7a9d0fe056 // indirect
 )
+
+go 1.13
```

Comment:
Latest opencensus-go client adds a constraint on go 1.13. If this is an issue, I'd be happy just updating `github.com/prometheus/client_golang`.